README.md

```diff
@@ -31,9 +31,15 @@ The OCM agent on the Managed cluster will see the ManifestWork on the Hub cluste
 - Hub cluster and remote/managed clusters need to have ArgoCD Application installed. See [ArgoCD website](https://argo-cd.readthedocs.io/en/stable/getting_started/) for more details.
 
 ## Getting Started
-1. Setup an OCM Hub cluster and registered an OCM Managed cluster.
+1. Setup an OCM Hub cluster and registered an OCM Managed cluster. See [Open Cluster Management Quick Start](https://open-cluster-management.io/getting-started/quick-start/) for more details.
+
+
 
 2. Install ArgoCD on both clusters.
+```
+kubectl create namespace argocd
+kubectl apply -n argocd -f https://raw.githubusercontent.com/argoproj/argo-cd/master/manifests/install.yaml
```

Comment:
This install the `latest` tag of ArgoCD which is generally not what users prefer. 
I think you can just tell them to visit the argocd website like above:
```
See [ArgoCD website](https://argo-cd.readthedocs.io/en/stable/getting_started/) for more details.
```

Reply:
Updated